XRDXRFutils/database.py

```diff
@@ -65,9 +68,28 @@ def get_theta(self, length = [1.541874], scale = [1.0], min_theta = None, max_th
                 intensity += [i * s]
             theta = concatenate(theta)
             intensity = concatenate(intensity) / 1000.0
+
+            # Merge peaks
+            if sigma is not None:
```

Comment:
Yes, I see your point. I was calling `sigma` the distance of merging because I intended to set it the same value as the sigma of synthetic spectrum. However, I agree that it is more clear to have separate names.

Now that parameter of `Phase.get_theta()` is called `distance_merge`.
Also, when creating instances of `GaussNewton`, `GammaSearch` and `GammaMap`, the two parameters `sigma` and `distance_merge` are kept separate.

See commit [3503cb9].(https://github.com/zpreisler/XRDXRFutils/pull/21/commits/3503cb9b036386fb92477c22aa547b4bf3d8e464)